stac_pydantic/api/extensions/query.py

```diff
@@ -1,7 +1,13 @@
+import operator
 from enum import auto
+from types import DynamicClassAttribute
+from typing import Any, Callable
 
 from stac_pydantic.utils import AutoValueEnum
 
+# TODO: These are defined in the spec but aren't currently implemented by the operator syntax
+UNSUPPORTED_OPERATORS = {"startsWith", "endsWith", "contains", "in"}
```

Comment:
these fields straight up don't work with python's `operator` 🤷‍♂️

Reply:
tbh I kind of hope CQL makes them irrelevent

Reply:
maybe we don't use the operator module 
```python
Operators = dict(
    eq = lambda x, y: x == y,
    ne = lambda x, y: x != y,
    lt = lambda x, y: x < y,
    le = lambda x, y: x <= y,
    gt = lambda x, y: x > y,
    ge = lambda x, y: x > y,
    startsWith = lambda x, y: x.startsWith(y),
    endsWith = lambda x, y: x.endsWith(y),
    constains = lambda x, y: y in x,
)
```